src/Core/src/Core/IFlyoutView.cs

```diff
@@ -11,6 +11,11 @@ public interface IFlyoutView : IView
 		/// </summary>
 		IView Flyout { get; }
 
+		/// <summary>
+		///  Gets the flyout custom icon.
+		/// </summary>
+		IImageSource FlyoutIcon { get; }
```

Comment:
This is a new API on the interface, so it is breaking. Maybe we can add it like this for net9, but maybe not.

Would a default implementation of null be OK? Or should this be a `IIconFlyoutView : IFlyoutVIew` to avoid breaks?

Reply:
Yeah null would be okay as in that case, the icon would behave as previously, so it would use a default drawer icon